### ISSUE TITLE
fromPromise swallows all errors. Consider rethrowing exception?

### ISSUE THREAD

**Issue by AriaFallah** [mobxjs/mobx-utils#6]
This would make it much easier to spot errors in your own code since they'll be output to the console. Additionally it lets the user of the utils decide how they want to handle errors themselves.

**Comment by mweststrate** [comment]
Could you elaborate on this one? See https://github.com/mobxjs/mobx-utils/blob/master/test/from-promise.js#L77, rejects or thrown errors should not be swallowed. Which promise library are you using?

**Comment by AriaFallah** [comment]
@mweststrate 

I'm using whatever babel / chrome provides. To be more clear, you're correct when you say that it rejects, but the error message is [nestled inside of the `reason` object](https://github.com/mobxjs/mobx-utils/blob/master/src/from-promise.ts#L24) instead of appearing in red text in the console.

Even more specifically, I got the MobX "an exception happened somewhere in your code...that isn't supposed to happen" error, and didn't know where it occurred initially until I thought to log the `fromPromise` object to look inside the `reason`.

Does that make sense?

TL;DR: Red text in console is gone, and located inside fromPromise.reason instead. This made my life harder.

**Comment by mweststrate** [comment]
Released 1.0.1, reject reason is now stored in `.value` (as well)
